Yank/utils.py

```diff
@@ -995,14 +957,14 @@ def process_unit_bearing_str(quantity_str, compatible_units):
 
 
 def to_unit_validator(compatible_units):
-    """Function generator to test unit bearing strings with Schema."""
+    """Function generator to test unit bearing strings with Cerberus."""
     def _to_unit_validator(quantity_str):
         return process_unit_bearing_str(quantity_str, compatible_units)
     return _to_unit_validator
 
 
 def generate_signature_schema(func, update_keys=None, exclude_keys=frozenset()):
-    """Generate a dictionary to test function signatures with Schema.
+    """Generate a dictionary to test function signatures with Cerberus' Schema.
```

Comment:
I guess we could move these methods in the new schema module now that we have one.

Reply:
I could do that

Reply:
I think im actually going to leave this here for now, It may be useful for others to have access to, and keeps with the minimal amount of refactoring needed